README.md

```diff
@@ -28,17 +34,23 @@ curl https://github.com/SketchUp/sketchup-ruby-debugger/releases/download/1.0.3.
 open -a /Applications/SketchUp\ 2015/SketchUp.app --args -rdebug "ide port=7000"
 ```
 
-Notes:
-- The port should match the remote debugger port setting configured in the IDE. Default port is 1234.
-- SketchUp will start up and appear to be frozen. It is waiting for the debugger to show up.
-- Launch remote debugging in the IDE, SketchUp should continue running. You should see breakpoints hit when Ruby code execution reaches the specified lines.
+## Command-line arguments:
+When launching SketchUp with debugging enabled, the following command-line arguments are supported:
+
+```
+-rdebug "ide [port=<number>] [wait]"
```

Comment:
Nice! I will update the Wiki page for RubyMine once this is merged and I've had some time to play with it.

The ability to disconnected sounded interesting. Will have to set this up with VSCode and see how that works.